### PR TITLE
refactor: use service api's in airtable import

### PR DESCRIPTION
**Pull request by dstala**
## Change Summary

Provide summary of changes with issue number if any.

## Change type

- [ ] feat: (new feature for the user, not a new feature for build script)
- [ ] fix: (bug fix for the user, not a fix to a build script)
- [ ] docs: (changes to the documentation)
- [ ] style: (formatting, missing semi colons, etc; no production code change)
- [ ] refactor: (refactoring production code, eg. renaming a variable)
- [ ] test: (adding missing tests, refactoring tests; no production code change)
- [ ] chore: (updating grunt tasks etc; no production code change)

## Test/ Verification

Provide summary of changes.

## Additional information / screenshots (optional)

Anything for maintainers to be made aware of

### CODE REVIEW

**Comment by pranavxc** [comment]
Now we can remove `NC_BASEURL_INTERNAL` environment variable completely.


https://github.com/nocodb/nocodb/blob/fdaf5ee272f7a9f812d5d401f234f61a2fdeae0d/packages/nocodb/src/lib/controllers/sync/import.ctl.ts#L108

**Comment by dstala** [comment]
> NC_BASEURL_INTERNAL

Done

File: packages/nocodb/src/lib/services/sync/helpers/readAndProcessData.ts [hunk 1]
```diff
@@ -291,12 +298,13 @@ export async function importLTARData({
...
-                assocMeta.modelMeta.id,
-                insertArray
-              );
+              const x = await bulkDataService.bulkDataInsert({
+                projectName: projectName,
+                tableName: assocMeta.modelMeta.title,
+                body: insertArray,
+                cookie: {},
+              });
+              console.log(x);
```
Comment (wingkwong) [inline packages/nocodb/src/lib/services/sync/helpers/readAndProcessData.ts:307]: remove console log

Comment (dstala) [reply packages/nocodb/src/lib/services/sync/helpers/readAndProcessData.ts]: > remove console log

Done

**Review by wingkwong — COMMENTED** [review]
1. please resolve the conflict
2. `projectName: projectName,` can be replaced by `projectName,`

**Review by wingkwong — APPROVED** [review]

**Pull request merged**